shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -43,6 +45,10 @@ - (UIScreen*)screen {
   return UIScreen.mainScreen;
 }
 
+- (MTLPixelFormat)pixelFormat {
+  return _pixelFormat;
```

Comment:
Do we need a new ivar?  Can't we use `self.layer.pixelFormat`?

Reply:
the layer ivar isn't exposed, this seemed more restricted.

Reply:
Right, you should keep your accessor.  Just use: 
```objc
- (MTLPixelFormat)pixelFormat {
  return self.layer.pixelFormat;
}
```

By duplicating this value we are opening ourselves up to bugs where the layer's pixel format does not match the FlutterView's pixel format.

Reply:
Fixed, I think!